R/intro_to_R/installing_R.Rmd

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
@benmbrew Why is this markdown here? It seems like it's not going to be used. Delete.